util/rss-cap.js

```diff
@@ -0,0 +1,49 @@
+require('require-self-ref')
+
+const FeedParser = require('feedparser')
+const request = require('request')
+const fs = require('fs')
+const Rx = require('rx') 
+const RxNode = require('rx-node')
+
+const rssDiff = require('~/src/rss-diff')
+const config = require('~/src/config')
+
+config.load()
+
+const PYPI_RSS_URL = config.getPypiRssUrl()
+
+// Observer that takes values and writes them to a file
+const writeObserver = Rx.Observer.create(
+    x => {
+        const json = JSON.stringify(x, null, '  ')
+        const datestring = (new Date().toISOString()) + '.json'
+        fs.writeFile(datestring, json, err => {
+            console.log(err)
+            return
+        })
+        console.log('Wrote rss feed out to: ' + datestring)
+    },
+    err => {
+        console.log(err)
+    },
+    () => {
+        console.log('Finished.')
+    }
+)
```

Comment:
Yep, it might end up being a Subject though which is an Observer and an Observable (if I want to handle events from the RabbitMQ like disconnects, etc).